src/store/story.js

```diff
@@ -114,20 +120,22 @@ export default {
       })
     },
     currentFrame: (state) => {
-      const f = state.frames.find(f => f.id === state.currentFrameId)
+      const isCoverFrame = state.currentFrameId === 0;
+      const f = isCoverFrame ? state.frames[0] : state.frames.find(f => f.id === state.currentFrameId)
       return {
-        text: f.text,
+        text: isCoverFrame ? state.storyTitle : f.text,
         style: makeFrameStyle(f),
         noImage: f.img === '',
         id: state.currentFrameId,
         imgAttribution: f.attribution,
-        imgTitle: f.imgTitle
+        imgTitle: isCoverFrame ? state.storyTitle : f.imgTitle
```

Comment:
I don't think this property is used at all, I can't find it.

It should be removed in a different PR.